samples/function_chaining_custom_status/.gitignore

```diff
@@ -0,0 +1,131 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
Same as above, I don't think we should push this, since this is managed by the VSCode extension 😄